code/game/objects/effects/portals.dm

```diff
@@ -101,7 +101,9 @@
 
 	if(prob(failchance))
 		icon_state = fail_icon
-		if(!do_teleport(M, locate(rand(5, world.maxx - 5), rand(5, world.maxy -5), 3), 0, bypass_area_flag = ignore_tele_proof_area_setting)) // Try to send them to deep space.
```

Comment:
Z3 used to be space so thats likely where this is from

Screw hardcoded levels.